tests/components/rfxtrx/test_config_flow.py

```diff
@@ -35,6 +35,16 @@ def com_port():
     return port
 
 
+async def init_hass_with_entry(hass, entry):
+    """Init hass with the entry under test."""
```

Comment:
We don't initialize `hass` so this name is not clear.
```suggestion
async def start_options_flow(hass, entry):
    """Start the options flow with the entry under test."""
```